2007-03-27  Gwenole Beauchesne  <gbeauchesne@mandriva.com>

	* hw/pc.c (INITRD_LOAD_ADDR): Increase max kernel size with
	initrd.

--- qemu-0.9.0/hw/pc.c.increase-initrd-load-addr	2007-03-15 13:36:53.000000000 -0400
+++ qemu-0.9.0/hw/pc.c	2007-03-27 13:27:43.000000000 -0400
@@ -37,7 +37,7 @@ extern int kvm_allowed;
 #define LINUX_BOOT_FILENAME "linux_boot.bin"
 
 #define KERNEL_LOAD_ADDR     0x00100000
-#define INITRD_LOAD_ADDR     0x00600000
+#define INITRD_LOAD_ADDR     0x00900000
 #define KERNEL_PARAMS_ADDR   0x00090000
 #define KERNEL_CMDLINE_ADDR  0x00099000
 
